Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -166,7 +162,7 @@ private let descriptionLabelStyle: LabelStyle = { (label: UILabel) in
   label
     |> \.text  %~ { _ in Strings.Kickstarter_is_not_a_store_Its_a_way_to_bring_creative_projects_to_life() }
     |> \.textColor .~ UIColor.ksr_text_dark_grey_500
-    |> \.font .~ UIFont.ksr_subhead(size: 12)
```

Comment:
This font seemed closer to the designs and allows text to fit for the default size.